packages/scandipwa/src/component/Field/Field.style.scss

```diff
@@ -268,6 +268,15 @@
                             word-break: break-word;
                             text-align: center;
                             border-color: var(--primary-success-color);
+
+                            & + button {
+                                border: 2px solid var(--primary-success-color);
+
+                                &:hover {
+                                    background-color: var(--primary-success-color);
+                                    color: #{$white};
```

Comment:
Why the $white variable doesn't have the # character so you should put it separately?

Reply:
Please put this color instead: `var(--color-white)`

Reply:
Please put this color: `var(--color-white)`